packages/fxa-settings/src/pages/InlineRecoverySetup/container.tsx

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { useMutation } from '@apollo/client';
+import { RouteComponentProps, useLocation, useNavigate } from '@reach/router';
+import LoadingSpinner from 'fxa-react/components/LoadingSpinner';
+import { useCallback, useEffect, useState } from 'react';
+import InlineRecoverySetup from './index';
+import { AuthUiErrors } from '../../lib/auth-errors/auth-errors';
+// import { useFinishOAuthFlowHandler } from '../../lib/oauth/hooks';
+import { getCode } from '../../lib/totp';
+import { MozServices } from '../../lib/types';
+import {
+  OAuthIntegration,
+  useAccount,
+  // useAuthClient,
+  useSession,
+} from '../../models';
+import { TotpToken } from '../InlineTotpSetup';
+import { VERIFY_TOTP_MUTATION } from './gql';
+
+export const InlineRecoverySetupContainer = ({
+  isSignedIn,
+  integration,
+  serviceName,
+}: {
+  isSignedIn: boolean;
+  integration: OAuthIntegration;
+  serviceName: MozServices;
+} & RouteComponentProps) => {
+  const account = useAccount();
+  const session = useSession();
+  const navigate = useNavigate();
+
+  // const authClient = useAuthClient();
```

Comment:
Remove?

Reply:
It's one of the arguments for `useFinishOAuthFlowHandler` but the linter didn't like unused vars.  🤷